### ISSUE TITLE
Incorrect styling

### ISSUE THREAD

**Issue by leeejune** [AlaskaAirlines/auro-accordion#136]
### Please describe the bug

1. Go to https://auro.alaskaair.com/components/auro/accordion/api
2. Look at accordion with chevron

### Reproducing the error on the docsite

None

### Expected behavior

1. Accordion should not have an outline around it (grey outline is applied right now)
2. Accordion should have a chevron on the right

# Current
![Screenshot 2024-10-24 at 5 19 20 AM](https://github.com/user-attachments/assets/1e8fd736-1c80-4201-90a5-f577d087421d)

# Expected
![Screenshot 2024-10-24 at 5 21 54 AM](https://github.com/user-attachments/assets/cc6617c3-be19-4127-8f54-f9bb4a53e287)

**Comment by jason-capsule42** [comment]
@leeejune please I've fixed the issue with the chevron not hiding but I am confused about the border color - that is only applied to accordion groups which I understood to be the intended design.

**Comment by leeejune** [comment]
> @leeejune I've fixed the issue with the chevron being hidden but I am confused about the border color - that is only applied to accordion groups which I understood to be the intended design.

Nope, the border is only meant to go on the bottom. You can see more detailed specs in the blueprint as well.
